octobot_trading/exchanges/util/exchange_market_status_fixer.pxd

```diff
@@ -16,25 +16,26 @@
 #  License along with this library.
 
 cpdef bint is_ms_valid(object value, bint zero_valid=*)
+cpdef bint check_market_status_limits(market_limit)
+cpdef bint check_market_status_values(market_limit, bint zero_valid=*)
 
 cdef void calculate_costs(dict market_limit)
 cdef void calculate_prices(dict market_limit)
 cdef void calculate_amounts(dict market_limit)
 cdef object get_markets_limit(dict market_limit)
 cdef void fix_market_status_limits_from_current_data(dict market_limit)
-cdef bint check_market_status_limits(market_limit)
-cdef bint check_market_status_values(market_limit, bint zero_valid=*)
 
 cdef class ExchangeMarketStatusFixer:
     cdef public object market_status
     cdef public object price_example
     cdef public object market_status_specific
 
-    cdef void __fix_market_status_precision(self)
-    cdef void __fix_market_status_limits(self)
-    cdef object __calculate_amount(self)
-    cdef void __fix_market_status_limits_with_price(self)
-    cdef double __get_price_precision(self)
-    cdef void __fix_market_status_precision_with_price(self)
-    cdef void __fix_market_status_precision_with_specific(self)
-    cdef void __fix_market_status_limits_with_specific(self)
+    cpdef double _get_price_precision(self)
+    cpdef void _fix_market_status_precision_with_price(self)
+    cpdef void _fix_market_status_limits_with_specific(self)
+    cpdef void _fix_market_status_limits_with_price(self)
```

Comment:
I had to make these cpdef to be able to call them in tests, what do you think about it @Herklos, should we keep cdef and avoid testing them directly ?

Reply:
We shouldn't change the cython header for test purposes. For similar cases I used to use CYTHON_IGNORE condition to pass the test when we are testing in compiled way.

Reply:
> We shouldn't change the cython header for test purposes. For similar cases I used to use CYTHON_IGNORE condition to pass the test when we are testing in compiled way.

To use CYTHON_IGNORE you need https://github.com/Drakkar-Software/OctoBot-Trading/pull/228